devtools/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
+
+services:
+  eventgridemulator:
+    image: eventgridemulator
+    build:
+      context: ../src
+      dockerfile: ../src/EventGridEmulator/Dockerfile
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=https://+:443;http://+:8080
+      - KESTREL__CERTIFICATES__DEFAULT__PATH=/etc/ssl/certs/localhost.crt
+      - KESTREL__CERTIFICATES__DEFAULT__KEYPATH=/etc/ssl/certs/localhost.key
+    ports:
+      - "6500:443"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
I'm curious if this was really required? I believed that docker containers could by default communicate with the host by using the `host.docker.internal` domain